R/data_modify.R

```diff
@@ -0,0 +1,219 @@
+#' Create new variables in a data frame
+#'
+#' Create new variables in a data frame. Unlike `base::transform()`, `data_modify()`
+#' can be used on grouped data frames, and newly created variables can be directly
+#' used.
+#'
+#' @param data The data frame, for which new variables should be created.
```

Comment:
```suggestion
#' @param data A data frame
```
just to be consistent with some other `data_` docs